doc/classes/AnimationTreePlayer.xml

```diff
@@ -610,6 +619,16 @@
 	</methods>
 	<members>
 		<member name="playback_process_mode" type="int" setter="set_animation_process_mode" getter="get_animation_process_mode" enum="AnimationTreePlayer.AnimationProcessMode">
+			The process notification in which to update animations. Default value: [enum ANIMATION_PROCESS_IDLE].
+		</member>
+		<member name="master_player" type="NodePath" setter="set_master_player" getter="get_master_player">
+			The [AnimationPlayer] node responsible for controlling this tree's animations.
```

Comment:
I think that actually the AnimationTreePlayer is controlling the AnimationPlayer not the other way around (not sure what's internal implementation, but that's the feeling that I have as a user). 
maybe instead something like: 
The [AnimationPlayer] node that provides animations for AnimationTreePlayer.